### DISCUSSION TITLE
I have intall your extension but it's not working on some link

### DISCUSSION THREAD

**Discussion by HB010** [FastForwardTeam/FastForward#703] [Q&A] [answered]
Link are : 

https://wlinki.cfd/?2a89c6f395=T1NIOFhXdFJNbmZrdHRYdkwzQkJrSHVud1Fib2t2MWpMdmV2V2JPTjJkL1RIZDRzZ21tUGZuTVdkMW5GVjh0QWMxZzlvdGxqZDFBM0NoMDQ4T2VVSytGWVByQkJQUHVDeHpLWlZGdHB6Qnc9

https://wlinki.cfd/3570/

**Comment by DumbAsFuck** [comment]
It won't work on all links, it depends on whether the server checks to make sure you waited.

**Comment by lostdusty** [comment, marked as answer]
We don't currently bypass this shortner.

**Comment by hieuck** [comment]
tôi từng sử dung Universal Bypass, thời gian đó tôi bỏ qua được liên kết nào và lần sau không gặp phải, bây giờ sử dụng lại FastForward thì không được như vậy nữa. Có phải tôi thực hiện thiếu "cái gì đó" không?
I used to use Universal Bypass, at that time I was able to bypass any links and never encountered them again, but now that I am using FastForward again, it doesn't work like that anymore. Am I missing something?

**Reply by lostdusty** [reply]
What is exactly not working?

**Reply by hieuck** [reply]
các liên kết tôi đã từng "bỏ qua" thì sau khi vào lại liên kết đó thì không tự "bỏ qua", điều này không giống như trước đây

the links I used to 'skip' after re-entering the link do not 'skip' itself, this is not the same as before

**Reply by lostdusty** [reply]
Which links?
